### DISCUSSION TITLE
Catch __search_all()__ errors

### DISCUSSION THREAD

**Discussion by iliion** [CS-SI/eodag#428] [Q&A]
Is it possible to catch errors thrown from __search_all()__  function?

**Comment by sbrunato** [comment]
Hello @iliion , [`search()`](https://eodag.readthedocs.io/en/latest/notebooks/api_user_guide/4_search.html#search()) method has a `raise_errors` parameter that can be set to true to make errors catchable, but [`search_all()`](https://eodag.readthedocs.io/en/latest/notebooks/api_user_guide/4_search.html#search_all()) has not.
Errors are logged but do not interrupt the pages iteration over search results.

**Reply by iliion** [reply]
Ok thanks, and I m terribly sorry to insist on this because i meant the `download_all()` function.  What if i want to be sure that every product is downloaded without errors?
